symcrypt-sys/src/bindings/x86_64_unknown_linux_gnu.rs

```diff
@@ -4788,9 +4793,14 @@ extern "C" {
 extern "C" {
     pub fn SymCryptHkdfSelfTest();
 }
+#[cfg(feature = "dynamic")]
```

Comment:
`SymCryptRandom` is not compiled unless `dynamic` feature is set. This applies to all target bindings